yeelight/flow.py

```diff
@@ -67,7 +71,7 @@ def as_list(self):
         :rtype: list
         """
         # Duration must be at least 50, otherwise there's an error.
-        return [max(50, self.duration), self._mode, self._value, self.brightness]
+        return [max(50, self.duration), self._mode, self._value, int(self.brightness)]
```

Comment:
Yeah, that feels a bit wrong here, but forcing the end app to cast (or try to figure out why it is not working, the error messaging from the bulb isn't very good..) doesn't feel right either.